google-cloud-billing/lib/google/cloud/billing.rb

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google-cloud-billing"
+
+module Google
+  module Cloud
+    ##
+    # Allows developers to manage billing for their Google Cloud Platform projects programmatically
```

Comment:
I'd suggest ending this sentence with a period.

Reply:
Done